docs/src/components/social-links-type.astro

```diff
@@ -1,7 +1,4 @@
 ---
-import { socialLinks } from '../../../packages/starlight/schemas/social';
-
-const socials = [...socialLinks].sort((a, b) => a.localeCompare(b, 'en'));
 ---
 
-<code>{`Partial<Record<${socials.map((social) => `'${social}'`).join(' | ')}, string>>`}</code>
+<code set:text="Array<{ label: string; icon: string; href: string }>" />
```

Comment:
I actually removed this component from the English docs now it no longer needs to generate this list. But as it’s still in use in other languages, I’ve updated the code here with a placeholder value until we can remove the component entirely.

Reply:
Should `icon` link to the icon page in the current locale like the English version does?

Reply:
Skipped before out of laziness, but done in 8fa2f66. Thanks for the suggestion!